docs/email_formatting.md

```diff
@@ -0,0 +1,73 @@
+# Email Formatting
```

Comment:
There wasn't really *any* guidelines on this as far as I could tell, so I wanted to put something here on the common gotchas.